src/store/model/spielegrotte.ts

```diff
@@ -0,0 +1,44 @@
+import {Store} from './store';
+
+export const Spielegrotte: Store = {
+	labels: {
+		inStock: [
+			{
+				container:
+					'html > body > table > tbody > tr > td > div > table > tbody > tr > td > center > table > tbody > tr > td > a.klein > img',
```

Comment:
I'm thinking this might be a better selector

```suggestion
					'img[src="neu_indenkorb_2.gif"]',
```

Reply:
Actually, scratch that. Let's keep it like the others.